### PR TITLE
LLM: using html to visualize the perf result for Arc

### PR DESCRIPTION
**Pull request by WeiguangHan**
using html to visualize the perf result for Arc, such as `http://10.239.44.136:8888/transformer_int4_gpu-results-2023-10-24.html`

### CODE REVIEW

File: .github/workflows/llm_performance_tests.yml [hunk 1]
```diff
@@ -129,3 +129,6 @@ jobs:
           export https_proxy=${HTTPS_PROXY}
           taskset -c 0-$((THREAD_NUM - 1)) python run.py
           curl -T ./*.csv ${LLM_FTP_URL}/llm/ggml-actions/perf/
+          cd ../../../test/benchmark
+          taskset -c 0-$((THREAD_NUM - 1)) python csv_to_html.py
```
Comment (hkvision) [inline .github/workflows/llm_performance_tests.yml:133]: no need taskset for this step?

File: .github/workflows/llm_performance_tests.yml [hunk 2]
```diff
@@ -129,3 +129,6 @@ jobs:
           export https_proxy=${HTTPS_PROXY}
           taskset -c 0-$((THREAD_NUM - 1)) python run.py
```
Comment (hkvision) [inline .github/workflows/llm_performance_tests.yml:130]: no need taskset -C?

File: .github/workflows/llm_performance_tests.yml (hunk 1 above)
Comment (WeiguangHan) [reply .github/workflows/llm_performance_tests.yml:133]: ok

File: .github/workflows/llm_performance_tests.yml (hunk 2 above)
Comment (WeiguangHan) [reply .github/workflows/llm_performance_tests.yml:130]: ok

File: python/llm/test/benchmark/csv_to_html.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Python program to convert 
+# CSV to HTML Table
+ 
+import os
+import pandas as pd
+
+folder_path = "../../dev/benchmark/all-in-one"
```
Comment (hkvision) [inline python/llm/test/benchmark/csv_to_html.py:22]: Better not hardcode the path, let's input the csv file as an argument?

Comment (WeiguangHan) [reply python/llm/test/benchmark/csv_to_html.py:22]: ok

**Review by hkvision — APPROVED** [review]

**Pull request merged**